condor/index.md

```diff
@@ -53,11 +53,12 @@ A detailed discussion of the Zug consensus may be found in the [Casper documenta
 In Casper 2.0, we are introducing a change to the execution engine which allows transactors to specify which of many possible VMs they wish to target with their transaction.
 
 ### CSPR Burn function
-In Casper 2.0 users can burn CSPR token ([CEP-92](https://github.com/casper-network/ceps/blob/master/text/0092-cspr-burn.md). This function is exposed as a `burn` function in the mint contract.
+In Casper 2.0 users can burn CSPR token [CEP-92](https://github.com/casper-network/ceps/blob/master/text/0092-cspr-burn.md). This function is exposed as a `burn` function in the mint contract.
```

Comment:
@devendran-m the typo was the omission of an `)` at the end. Please update to have parentheses around the word CEP-92. Thanks.